build.sbt

```diff
@@ -86,6 +86,7 @@ lazy val prinz = (project in file("prinz"))
         "com.typesafe" % "config" % typesafeConfigV,
         "org.scala-lang" % "scala-reflect" % scalaV,
         "ch.qos.logback" % "logback-classic" % logbackV,
+        "org.jsoup" % "jsoup" % jsoupV,
```

Comment:
remove from pmml dependencies

Reply:
for now it has to be here, cause HTTPRepositoryClient is in pmml package. after merging with your commit, it will be removed